Release 3.0.0 (#3429)
nginxinc/kubernetes-ingress: docs/content/technical-specifications.md

We advise users to run the most recent release of the NGINX Ingress Controller, and we issue software updates to the most recent release. We provide technical support for F5 customers who are using the most recent version of the NGINX Ingress Controller, and any version released within two years of the current release.

> The current release version is 3.x supports the discovery.k8s.io/v1 API version of EndpointSlice, available from Kubernetes 1.21 and later.

Reviewer: ```suggestion
The current release version 3.x supports `discovery.k8s.io/v1` API version of EndpointSlice, available from Kubernetes 1.21 onwards.
```

{{% table %}}
| NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
| --- | --- | --- | --- | --- |

> | 3.0.0 | 1.25 - 1.19 | 0.15.2 | 1.2.1 | 1.23.2 / R28 |

Reviewer 2: This shouldn't delete `2.4.2`

{{% table %}}
|Name | Base image | Third-party modules | DockerHub image | Architectures |
| ---| ---| ---| --- | --- |

> |Alpine-based image | ``nginx:1.23.2-alpine``, which is based on ``alpine:3.16`` | NGINX OpenTracing module, OpenTracing library, OpenTracing tracers for Jaeger, Zipkin and Datadog | ``nginx/nginx-ingress:3.0.0-alpine`` | arm/v7, arm64, amd64, ppc64le, s390x |
> |Debian-based image | ``nginx:1.23.2``, which is based on ``debian:11-slim`` | NGINX OpenTracing module, OpenTracing library, OpenTracing tracers for Jaeger, Zipkin and Datadog | ``nginx/nginx-ingress:3.0.0`` | arm/v7, arm64, amd64, ppc64le, s390x |
> |Ubi-based image | ``nginxcontrib/nginx:1.23.2-ubi``, which is based on ``redhat/ubi9-minimal`` |  | ``nginx/nginx-ingress:3.0.0-ubi`` | arm64, amd64, ppc64le, s390x |

Reviewer 2: ```suggestion
|Alpine-based image | ``nginx:1.23.3-alpine``, which is based on ``alpine:3.17`` | NGINX OpenTracing module, OpenTracing library, OpenTracing tracers for Jaeger, Zipkin and Datadog | ``nginx/nginx-ingress:3.0.0-alpine`` | arm/v7, arm64, amd64, ppc64le, s390x |
|Debian-based image | ``nginx:1.23.3``, which is based on ``debian:11-slim`` | NGINX OpenTracing module, OpenTracing library, OpenTracing tracers for Jaeger, Zipkin and Datadog | ``nginx/nginx-ingress:3.0.0`` | arm/v7, arm64, amd64, ppc64le, s390x |
|Ubi-based image | ``nginxcontrib/nginx:1.23.3-ubi``, which is based on ``redhat/ubi9-minimal`` |  | ``nginx/nginx-ingress:3.0.0-ubi`` | arm64, amd64, ppc64le, s390x |
```

{{% table %}}
|Name | Base image | Third-party modules | F5 Container Registry Image | Architectures |
| ---| ---| --- | --- | --- |

> |Alpine-based image | ``alpine:3.16`` | NGINX Plus JavaScript and OpenTracing modules, OpenTracing tracers for Jaeger, Zipkin and Datadog | `nginx-ic/nginx-plus-ingress:3.0.0-alpine` | arm64, amd64 |

Reviewer 2: ```suggestion
|Alpine-based image | ``alpine:3.17`` | NGINX Plus JavaScript and OpenTracing modules, OpenTracing tracers for Jaeger, Zipkin and Datadog | `nginx-ic/nginx-plus-ingress:3.0.0-alpine` | arm64, amd64 |
```
